lib/bible_ref/languages/english.rb

```diff
@@ -88,7 +95,7 @@ def books
           '1JN' => { match: /^1 ?jo?h?n/,        name: '1 John'                 },
           '2JN' => { match: /^2 ?jo?h?n/,        name: '2 John'                 },
           '3JN' => { match: /^3 ?jo?h?n/,        name: '3 John'                 },
-          'JUD' => { match: /^(jud$|jd$|jude$)/, name: 'Jude'                   },
+          'JUD' => { match: /^(jud|jd|jude)/,    name: 'Jude'                   },
```

Comment:
I'm a bit nervous about this change, given it was just fixed in #7. Can you make sure that the existing behavior of differentiating between Judith and Jude is left intact. (If not, we may need a regression test to keep it from happening in the future.)

Reply:
Sure.  I will write a test to validate it. 🤓👍